documentation/src/main/asciidoc/user_guide/transactions.adoc

```diff
@@ -276,7 +262,27 @@ Or
 new ConfigurationBuilder().versioning().scheme(SIMPLE);
 ----
 
-NOTE: `SIMPLE` versioning is an implementation of the proposed `EntryVersion` interface, backed by a long that is incremented each time the entry is updated.
+==== Forcing write locks on keys in pessimitic transactions
+
+Write-skews checks do not occur with pessimistic transactions. To avoid write-skews with pessimistic transactions, lock keys at read-time with `Flag.FORCE_WRITE_LOCK`. For example,
+
+[source, java]
+----
+if (!cache.getAdvancedCache().lock(key)) {
```

Comment:
I wonder if we should mention this has to be in an ongoing transaction?

Reply:
@wburns the "FORCE_WRITE_LOCK" flag can be used with ongoing transactions? so the transaction cannot be complete, aborted, or some other state?